string msg
float64 angle
int32 circle_state
bool scan_trigger
bool apply_config
bool end_state
bool scan_3d
bool full_scan
bool robot_mode
bool oct_mode
bool octa_mode
bool oce_mode

